packages/suite/src/views/wallet/anonymize/components/CoinjoinCustomStrategy.tsx

```diff
@@ -99,15 +82,13 @@ interface CoinjoinCustomStrategyProps {
 
 export const CoinjoinCustomStrategy = ({
```

Comment:
That whole component is not used anymore 🤔

Reply:
Was this left out on purpose? 
<img width="670" alt="image" src="https://user-images.githubusercontent.com/45338719/217090643-0928cdf1-3627-46fd-a713-249a51546bda.png">

Reply:
Custom setup has been removed temporarily. We're not yet sure whether we implemented as in #7431 or move the settings to the Account tab.

Reply:
I just add it as a task to that issue, not related to this one.